src/docs/metrics/amount_in_top_holders/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Amount in Top Holders
```

Comment:
use `-` instead underscore in file title:
src/docs/metrics/amount_in_top_holders/index.md